selfservice/flow/registration/flow.go

```diff
@@ -90,6 +90,14 @@ func NewFlow(exp time.Duration, csrf string, r *http.Request, ft flow.Type) *Flo
 	}
 }
 
+func (f *Flow) GetType() flow.Type {
+	return f.Type
+}
+
+func (f *Flow) GetRequestURL() string {
```

Comment:
These new methods need tests

Reply:
Sounds good. should I be adding unit tests for the method on each implementing type?